packages/sdk/src/swap/sdk.ts

```diff
@@ -153,6 +158,14 @@ export class SwapSDK {
         channelOpeningFee: result.channelOpeningFee,
       };
     } else {
+      assert(
+        !depositAddressRequest.brokerCommissionBps,
+        'Broker commission is only supported only when initializing the SDK with a brokerUrl',
+      );
```

Comment:
this is a breaking change for people who might instantiate the sdk with a commission but use our sdk. i would add an extra change to changelog saying that this will now throw an error instead of silently ignoring

Reply:
wanted to add this but then i noticed that it wasnt possible to set a commission without providing a broker url before (you could only instantiate the sdk with a url + commission)

Reply:
cool, nvm then